release-notes/opensearch-neural-search.release-notes-2.18.0.0.md

```diff
@@ -3,6 +3,8 @@
 
 Compatible with OpenSearch 2.18.0
 
+### Bug Fixes
+- Fixed incorrect document order for nested aggregations in hybrid query ([#956](https://github.com/opensearch-project/neural-search/pull/956))
```

Comment:
put in release notes file directly because notes were pulled from CHANGELOG for 2.18 release